blogs/2021/01/ext-bisect.md

```diff
@@ -0,0 +1,40 @@
+# “Just like git-bisect, but for VS Code extensions.”
+
+The true power of VS Code is its extensions: theme extensions add colors and icons, language extensions provide IntelliSense and enable code navigation, debugger extensions enable you to drill into bugs. There are extensions that play music, some that show stock tickers, and there are extensions that enable collaborative work across locations and time zones. The VS Code marketplace hosts more than 23K extensions and it is not uncommon that users have 50+ extensions installed. With so many extensions bugs are inevitable and instead of denial we want to make trouble shooting easy.
+
+We love extensions. There are no bad extensions. Like all software, extensions have bugs and feature gaps and for reading ease and artificial drama, let’s use “bad extensions”. Understand it is an extension showing unwanted behavior. In the fact, we have designed VS Code with bad extensions in mind and therefore run them in a separate [process]( https://code.visualstudio.com/api/advanced-topics/extension-host). The guarantee is that the cursor always blinks and that you can always save your work. However, not all bad extensions crash or freeze a process, sometimes it is just a small, sporadically showing, annoyance.
```

Comment:
We love extensions and don't really think there are any "bad" extensions. However, like all software, extensions...